tutorials/detect.ipynb

```diff
@@ -181,9 +244,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Measurements\n",
+    "## Li & Ma significance maps\n",
```

Comment:
Maybe here, the interesting point is rather to go beyond the point source assumption that is made by the `TSMapEstimator`

Reply:
I added a transition comment at the end of the TS map part about extended sources:

> Note that we used the instrument point-spread-function (PSF) as kernel, so the hypothesis we test is the presence of a point source. In order to test for extended sources we would have to use as kernel an extended template convolved by the PSF. Alternatively, we can compute the significance of an extended excess using the Li & Ma formalism, which is faster as no fitting is involve.

Reply:
Perfect